### ISSUE TITLE
Error During the common_startup event phase

### ISSUE THREAD

**Issue by SheReyy** [yuoft/Endless#12]
Mc: 1.16.5
Forge: 36.2.33
EndlessFake: Endless-1.16.5-1.3.3.jar

After updating the mod, I got this error:
![image](https://user-images.githubusercontent.com/91336389/177631938-791f8da7-0cdb-4c20-97fb-03714bc2c409.png)
[Here](https://gist.github.com/IamRyan30/d682b1fe1693dd4f068678a87e90909d)'s the latest.log.

嗨，更新mod後，我得到了這個錯誤。還有最新的日誌

**Comment by yuoft** [comment]
我并没有发现错误，但是我看到你使用的forge版本较低，我使用的是36.2.34版本。
I didn't find any errors, but I saw that you used a lower version of forge, and I used version 36.2.34.

**Comment by IamRyan30** [comment]
Got it. I'll look into that, thank you.

知道了。我研究一下，謝謝

**Comment by IamRyan30** [comment]
> 我并没有发现错误，但是我看到你使用的forge版本较低，我使用的是36.2.34版本。 I didn't find any errors, but I saw that you used a lower version of forge, and I used version 36.2.34.

I've updated to forge 36.2.34 and to no success. I've also found this in the log which had the crash:
![image](https://user-images.githubusercontent.com/91336389/178145877-73edfa67-2d75-4752-8f84-ec9b69754098.png)

我已經更新到偽造 36.2.34 並且沒有成功。我還在發生崩潰的日誌中發現了這一點

**Comment by yuoft** [comment]
你是否加载了这个模组？obfuscate
Have you loaded this module? obfuscate。

**Comment by IamRyan30** [comment]
Yes. It's required by.. A space mod, I think.
是的。它是由.. 需要的。我想是一個太空模組。

**Comment by IamRyan30** [comment]
> Yes. It's required by.. A space mod, I think. 是的。它是由.. 需要的。我想是一個太空模組。

Yeah, it's required by this one: https://www.curseforge.com/minecraft/mc-mods/beyond-earth/files/3557337
是的，這是這個要求
